### ISSUE TITLE
Radar 3-bar Settings Menu GONE

### ISSUE THREAD

**Issue by flynfrank** [ELY3M/wX---modded-by-ELY-M#7]
I believe this same issue took place the last time I mentioned being a somewhat similar issue in the past. Using an Android phone, on the most recent Update #74 the Settings Menu option located in the NEXRAD radar viewing page no longer exists. Also, The Navigation Menu on the Home Page no longer exists. It appears a "+" (plus) icon has replaced the Navigation Bar. However, we are now not able to get to ALL of the WX app's settings features. And, in almost all cases, once you leave the Radar Page, there is NO navigation to return to that page. The only solution for me is to reboot the phone. 
The bottom of my Radar Page now displays 5-Radar Layer icons, a Play icon, and a Star. 
This has always been the most badass weather app in my opinion. Thank you so much for your dedication to this project!

**Comment by ELY3M** [comment]
Thank you for your bug report. I will look into this.

**Comment by ELY3M** [comment]
I know why the 3-bar setting icon was missing.  as I added two new icons in the menu.  the last icon got pushed.  
I made decision to get rid of the earth icon that goes to the ugly map.  I never used that section.  I just touch the radar map in exact place and go to exact radar.

**Comment by ELY3M** [comment]
release #75 will be up in one min.

**Comment by flynfrank** [comment]
Ok great. Thanks for looking into this issue. I use this app a great deal. I brag to others and refer to it as the best weather app easily. 

I’ve been close to contacting you about adding a feature to the Wx app. Basically, what I would like to see is either a “Left/Right” arrow (like this < > ) to be used to pause Radar Animation and allowing each “NEXRAD Frame” to be scrolled in the user’s choice of forward or backward direction. Or, if not the mentioned arrows, then add a Finger Scroll/Navigation bar. I personally use this technique on the College of Dupage website. It allows you to better see Weather Manipulation. Adding this feature to the Wx app would better allow me to see Storm Growth and Cloud Dissipation Characteristics in each weather system.

Oh btw, just sayin, but in the User Interface Settings/Theme/ Color BlackAqua, on the Dark Mode Background should be listed as the Default Color. That combo in my opinion is badass! 👍

One last question… The 2 new icons added to the NEXRAD Radar page which are referred as NOQ: Base Refectivity Super-res AND NOU: BASE Velocity Super-res. Neither of these icons display a map. I have tried using both on old NEXRAD locations as well as locations that have been upgraded to the most advanced NEXRAD locations here in the United States. I have yet to see if I’m able to change the icon configurations. And yes, I have tried repeatedly pressing the icon to see if will scroll like to 1st icon which has 2 options.

I’m now going to work on adding the fix update you added. Thanks again!

Frank

Stay Safe!
flyNfrank
